src/scheduler/ExecutableIntervalSchedule.ts

```diff
@@ -0,0 +1,65 @@
+import humanInterval from 'human-interval';
+import { max } from 'lodash';
+import { DateTime } from 'luxon';
+import { IntervalSchedule } from '../job/MomoJob';
+import { momoError } from '../logging/error/MomoError';
+import { TimeoutHandle, setSafeIntervalWithDelay } from '../timeout/setSafeIntervalWithDelay';
+import { Logger } from '../logging/Logger';
+import { ExecutableSchedule, NextExecutionTime } from './ExecutableSchedule';
+import { ExecutionInfo } from '../job/ExecutionInfo';
+
+export class ExecutableIntervalSchedule implements ExecutableSchedule<IntervalSchedule> {
```

Comment:
Can we have some unit tests for the Executable***Schedule classes?

Reply:
Sure. I'll add unit tests that check the simple methods and the `nextExecution` calculation. The actual running of the jobs is well tested in the integration tests and the `JobScheduler.spec.ts`.